kapacitor/vars.go

```diff
@@ -39,15 +40,16 @@ func Vars(rule chronograf.AlertRule) (string, error) {
 		%s
         var crit = %s
  `
-			return fmt.Sprintf(vars,
-				common,
-				rule.TriggerValues.Value), nil
+			// If critical value is a string, we'll
+			// need to single-quote it.
+			crit := critVar(rule.TriggerValues.Value)
```

Comment:
maybe `critVar` could be named `quote` or `formatQuote` or something? I feel like the comment above wants a better name out of this function :) You could potentially inline it too with a different name:

```go
return fmt.Sprintf(vars, common, formatQuote(rule.TriggerValues.Value))
```